src/commands/codepush/deployment/list.ts

```diff
@@ -22,7 +28,17 @@ export default class CodePushDeploymentListListCommand extends AppCommand {
       const httpRequest = await out.progress("Getting CodePush deployments...", clientRequest<models.Deployment[]>(
         (cb) => client.codePushDeployments.list(app.ownerName, app.appName, cb)));
       deployments = httpRequest.result;
-      out.table(out.getCommandOutputTableOptions(["Name", "Key"]), deployments.map((deployment) => [deployment.name, deployment.key]));
+
+      if (this.displayKeys) {
```

Comment:
If we're defaulting this to `false` and changing the default behaviour of the command - can we add some help text to the default output to guide users who may already be using the `Name, Key` table output?

That way users can quickly see what they need to add to the command to see the output they were using previously.

Reply:
Yeah, It would be better. Added text before table. Thanks